### PR TITLE
Ng storybook

### PR DESCRIPTION
**Pull request by GuillaumeNury**
First batch of ng stories

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1612

File: stories/documentation/users/select/user-homonyms.stories.ts [hunk 1]
```diff
@@ -81,5 +39,49 @@ const template: Story<UserHomonymsStory> = (args: UserHomonymsStory) => ({
 	props: args,
 });
 
-export const basic = template.bind({});
-basic.args = {};
+const code = () => `
+/* Afin de créer une sélection d'utilisateur custom avec gestion des homonymes */
```
Comment (shajz) [inline stories/documentation/users/select/user-homonyms.stories.ts:43]: Shouldn't code comments be written in english?

File: stories/documentation/overlays/popover/popover.stories.ts [hunk 2]
```diff
@@ -63,25 +55,31 @@ const template: Story<PopoverStory> = (args: PopoverStory) => ({
...
-	alignment: 'center',
-	trigger: 'click',
-}
+basic.parameters = {
+	// Disable controls as they are not modifiable because of ComponentWrapper
+	controls: { include: [] },
+	docs: {
+		source: {
+			language: 'ts',
+			code: `
```
Comment (shajz) [inline stories/documentation/overlays/popover/popover.stories.ts:64]: It'd be nice to write this code block the same way (inline or via function calls) in all sb files

**Review by shajz — APPROVED** [review]

**Review by jeremie-lucca — DISMISSED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1612

**Review by jeremie-lucca — APPROVED** [review]

**Pull request merged**